glide.lock

```diff
@@ -357,39 +387,4 @@ imports:
   - tools/clientcmd/api/v1
   - tools/metrics
   - transport
-testImports:
```

Comment:
I think you have a bad version of glide. You shouldn't be removing this whole testImports block from the glide.lock